src/main/java/net/glowstone/boss/BossBarManager.java

```diff
@@ -1,83 +1,49 @@
 package net.glowstone.boss;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.UUID;
-import net.glowstone.GlowServer;
+import java.util.Collection;
+import net.glowstone.entity.GlowPlayer;
+import org.bukkit.boss.BossBar;
 import org.bukkit.entity.Player;
 
-public class BossBarManager {
-
-    private static BossBarManager instance;
-
-    private final List<GlowBossBar> bossBars = new ArrayList<>();
-    private final GlowServer server;
-
-    /**
-     * Create an instance for the given server.
-     * @param server the server
-     */
-    public BossBarManager(GlowServer server) {
-        if (instance != null) {
-            throw new RuntimeException("BossBar Manager has already been initialized.");
-        }
-        this.server = server;
-        instance = this;
-    }
-
-    public static BossBarManager getInstance() {
-        return instance;
-    }
-
+public enum BossBarManager {
```

Comment:
Please make this a `static class`.

Reply:
Done; some authors recommend that utility classes that aren't going to be instantiated be zero-instance enums.